.github/workflows/buildAndTest.yml

```diff
@@ -24,6 +24,11 @@ concurrency:
 # Use Cached LLVM to improve build times.
 jobs:
   build-test:
+    env:
+      LLVM_PROJ_DIR: "llvm-project"
+      LLVM_BUILD_DIR: "llvm-build"
+      STABLEHLO_BUILD_DIR: "stablehlo-build"
+      STABLEHLO_API_BUILD_DIR: "stablehlo-api-build"
```

Comment:
Perhaps we can call this "stablehlo-python-build" (and rename related things accordingly), given that the focus of this build is on Python, not on the API? (We also have C and C++ APIs which are already provided by the regular build).